File: src/modules/Booking&SeatManagement/views/ConsoleView.js
class ConsoleView {
    static displaySeats(seats) {
        console.log("\nSeat Map:");
        Object.values(seats).forEach(seat => {
            console.log(`${seat.seatNumber}: ${seat.status}`);
        });
    }

    static showMessage(message) {
        console.log(message);
    }

    static showBooking(booking) {
        console.log("\nBooking Successful!");
        console.log(`Reference: ${booking.bookingReference}`);
        console.log(`Seats: ${booking.seats.join(', ')}`);
    }
}

module.exports = ConsoleView;

File: src/modules/Booking&Seat Module/view/ConsoleView.java

package view;

import model.Seat;
import java.util.Map;

public class ConsoleView {

    public static void showSeats(Map<String, Seat> seats){
        System.out.println("\nSeats:");
        for(Seat s: seats.values()){
            System.out.println(s.seatNumber + " : " + s.status);
        }
    }

    public static void message(String msg){
        System.out.println(msg);
    }
}
